doc/build_example.py

```diff
@@ -6387,6 +6407,9 @@ def operator_unary(row: sf.Series) -> tp.Iterator[str]:
         if attr == '__abs__()':
             yield f'bt = {icls}({kwa(BATCH_INIT_B)})'
             yield f'abs(bt).to_frame()'
+        elif attr == 'abs()':
+            yield f'bt = {icls}({kwa(BATCH_INIT_B)})'
+            yield f'bt.abs()'
```

Comment:
To show the result of using the `abs()` here, we need to call `to_frame()`, so:
```suggestion
            yield f'bt.abs().to_frame()'
```